bedrock/mozorg/templates/mozorg/contribute/friends.html

```diff
@@ -0,0 +1,64 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contribute/contribute-base.html" %}
+
+{% block page_title %}
+    {{ _('Join Firefox Friends today!') }}
+{% endblock %}
+
+{% block body_class %}{{ super() }} firefox-friends{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'contribute-friends' %}
+{% endblock %}
+
+{% block js %}
+{% endblock %}
+
+{% block contrib_page_title %}{{ _('Join Firefox Friends today!') }}{% endblock %}
+
+{% block alt_header %}
+<img src="{{ static('img/contribute/friends/firefox-friends-logo.png') }}" width="186" class="ff-logo" alt="{{_('Firefox Friends logo')}}" />
+<div class="ff-header-content">
+  <h1>{{_('Join Firefox Friends today!')}}</h1>
```

Comment:
Yeah, I agree "How it works" shouldn't be the `<h1>`. I just wondered that "Join Firefox Friends today!" feels like it belongs to the main contents of the document (whereas `#masthead` and `#colophon` belong to the body). But this is all minor semantics, do what you feel is best here :)